pages/apis/rest_api/clusters.md

```diff
@@ -0,0 +1,86 @@
+# Clusters API
+
+## Cluster data model
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>id</code></th><td>ID of the cluster</td></tr>
+  <tr><th><code>graphql_id</code></th><td><a href="/docs/apis/graphql-api#graphql-ids">GraphQL ID of the cluster</a></td></tr>
+  <tr><th><code>name</code></th><td>Name of the cluster</td></tr>
+  <tr><th><code>description</code></th><td>Description of the cluster</td></tr>
+  <tr><th><code>emoji</code></th><td>Emoji that represents the cluster</td></tr>
+  <tr><th><code>color</code></th><td>Color that represents the cluster</td></tr>
```

Comment:
I wonder if "represents the cluster" is too deep 🤔

Would "Emoji assigned to the cluster" or "Emoji set for the cluster" be better?